cloud/src/definer/crud.definer.js

```diff
@@ -119,16 +119,16 @@ Parse.Cloud.define('postObjectsToClass', (request) => new Promise((resolve, reje
 
   // add key/value from the local object to SurveyPoint
   const { localObject } = request.params;
-  for (const key in localObject) {
+  localObject.forEach((key) => {
```

Comment:
@jamccomb92 you'll need to find a different way to iterate through this object, this should break

Reply:
yea this unfortunately won't work

Error: {"message":"localObject.forEach is not a function","code":141}

you'll need to find a way to iterate through keys in an object

`for ... in...` is specifically for object iteration
`forEach`  is for array iteration

Reply:
This has been fixed on most recent PLATFORM-110 PR: https://github.com/hopetambala/puente-node-cloudcode/pull/396 
This PR can be closed unless we want to continue using the searchIndex portion